getting_started/scripting/c_sharp/c_sharp_differences.rst

```diff
@@ -25,14 +25,14 @@ Global functions like ``print``, ``var2str`` and ``weakref`` are located under
 Math
 ----
 
-Math functions like ``abs``, ``acos``, ``asin``, ``atan`` and ``atan2`` are
+Math functions, like ``abs``, ``acos``, ``asin``, ``atan`` and ``atan2`` are
```

Comment:
Shouldn't there be a comma at the end of the enumeration? (Same for the "Random" section.)

Reply:
> Shouldn't there be a comma at the end of the enumeration? (Same for the "Random" section.)

@akien-mga : But [of course](https://en.wiktionary.org/wiki/such_as)! You're making me look bad. :laughing: Thank you for keeping an eye out for what I miss. :smile: